Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -87,18 +89,23 @@ try {
         }
     }
 
+    Write-Host "OUTPUTS:"
     $outSettingsJson = $outSettings | ConvertTo-Json -Depth 99 -Compress
-    Add-Content -Path $env:GITHUB_OUTPUT -Value "SettingsJson=$outSettingsJson"
-    Add-Content -Path $env:GITHUB_ENV -Value "Settings=$OutSettingsJson"
-    Write-Host "SettingsJson=$outSettingsJson"
+    if ($useBase64) {
+        Add-Content -Path $env:GITHUB_OUTPUT -Value "SettingsJson=$([Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($outSettingsJson)))"
```

Comment:
Isn't the actual problem that we output every setting (line 85 in this file: `Add-Content -Path $env:GITHUB_ENV -Value "$setting=$($settingValue | ConvertTo-Json -Depth 99 -Compress)`).

So, with that change, every setting will be in `$env:GITHUB_ENV` and all the settings (JSON-formatted and maybe base64-encoded) will the in the `$env:GITHUB_OUTPUT`. 

I'm not convinced the issue will be solved.

Reply:
The real problem is that settings where as individual values + as a json construct 2 or 3 times.
This PR removes the json constructs.
It might be a good idea to generally look at what ENV variables are needed - but for now, this fixes the issue (I am convinced:-)) and we can fix the other ones later,

Reply:
This is a build of ALAppExtensions with the new ENV mechanism: